.npmignore

```diff
@@ -16,3 +20,5 @@ npm-debug.log
 node_modules
 coverage
 coverage-e2e
+
+test/
```

Comment:
New line should be at the end of the file - sorry to nitpick!

Also, can you please squash your `updated dependency path` commits into one?